README.md

```diff
@@ -581,3 +585,13 @@ the entry you can expand it to see **resource.labels.pod_id**.
 ## Contributing
 
 Please refer to the [developer_guide](developer_guide.md)
+
+## Community
+
+* [Slack Channel](https://join.slack.com/t/kubeflow/shared_invite/enQtMjgyMzMxNDgyMTQ5LWUwMTIxNmZlZTk2NGU0MmFiNDE4YWJiMzFiOGNkZGZjZmRlNTExNmUwMmQ2NzMwYzk5YzQxOWQyODBlZGY2OTg)
```

Comment:
Should we add this to kubeflow/community and just link to it from here? This way we don't have to duplicate it for every repo.

Reply:
SGTM, I will update it.